README.md

```diff
@@ -163,3 +163,28 @@ test('it deletes an item', async function(assert) {
   assert.ok(stub.calledWith('Are you sure?'));
 });
 ``` 
+
+### Mocking `open()`
+
+Here is an example that assigns a new function to replace `open`.
+You can check if the function has been called as well as the value of its parameters.
+
+```js
+import { module, test } from 'qunit';
+import { click, visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { default as window, reset } from 'ember-window-mock';
+
+module('Acceptance | new window', function(hooks) {
+  setupApplicationTest(hooks);
+
+  test('it opens a new window when clicking the button', async function(assert) {
+    await visit('/');
+    window.open = (urlToOpen) => {
+      assert.equal(urlToOpen, 'http://www.example.com/some-file.jpg');
+    };
+    await click('button');
+    reset();
```

Comment:
It's better to use the test setup helper instead of calling `reset()` in the test itself (like when the test fails, `reset()` might not get called). See https://github.com/kaliber5/ember-window-mock#resetting-the-state-with-the-new-testing-apis 

Btw, I need to update the other examples with this as well...

Reply:
Updated. Sorry for the late answer.